src/transformers/convert_reformer_trax_checkpoint_to_pytorch.py

```diff
@@ -0,0 +1,211 @@
+# coding=utf-8
```

Comment:
This conversion is quite specific to the `crime-and-punishment` model, but it's the only one we have currently. Might rename the file at a later stage or adapt it to be more flexible to future new models from trax.